src/sentry/db/postgres/base.py

```diff
@@ -93,7 +93,7 @@ def executemany(self, sql, paramlist=()):
 
 class DatabaseWrapper(DjangoDatabaseWrapper):
     SchemaEditorClass = DatabaseSchemaEditorProxy
-    queries_limit = 10000
```

Comment:
Is this limit something we're hitting? Is this intentional?

Reply:
It just limited how much output the DB can report. For very large tests, we exceed that output (due to adding more users, etc), so I bumped it a bit.